test/built-ins/Temporal/PlainDateTime/prototype/toString/calendarname-wrong-type.js

```diff
@@ -24,3 +24,12 @@ TemporalHelpers.checkStringOptionWrongType("calendarName", "auto",
   (calendarName) => datetime.toString({ calendarName }),
   (result, descr) => assert.sameValue(result, "2000-05-02T12:34:56.987654321[u-ca=custom]", descr),
 );
+
+const invalidCals = [false, 3, null];
```

Comment:
Aren't these covered yet, though?

Reply:
I'm not sure I understand. Do you mean that these checks duplicate existing tests? These non-string values used to (mistakenly) live in argument-string-invalid.js, but I extended this (existing) test with additional bad values.